pixutils/formats/pixelformats.py

```diff
@@ -468,6 +531,38 @@ def get_formats():
         ( ( 4, 2 ), ),
     )
 
+    SBGGR12P = PixelFormat('SBGGR12P',
+        None, 'pBCC',
+        PixelColorEncoding.RAW,
+        True,
+        ( 2, 2 ),
+        ( ( 6, 4 ), ),
```

Comment:
These should be ( 3, 2 ). Or do you think line widths with odd number of bytes is not ok?